Makefile

```diff
@@ -232,15 +231,19 @@ test-upgrade-e2e: kind-cluster run-latest-release image-registry pre-upgrade-set
 e2e-coverage:
 	COVERAGE_OUTPUT=./coverage/e2e.out ./hack/test/e2e-coverage.sh
 
-.PHONY: kind-load
-kind-load: $(KIND) #EXHELP Loads the currently constructed image onto the cluster.
+kind-load: $(KIND) build-catalogd #EXHELP Loads the currently constructed images into the KIND cluster.
```

Comment:
Since `kind-load` didn't already depend on build-operator-controller (or whatever it was called), I wouldn't expect it to now depend on `build-catalogd`

Maybe the `run` target should be the thing that depends on `build-catalogd` since that is where `docker-build` is depended upon?

Reply:
I see your POV. Good catcher.
It is added already at: docker-build: build-linux build-catalogd
(where we called the build for con-oper)

So, all should still working by just reverting this one.

Reply:
Done, So I am closing this one, but feel free to re-open if you need to.